src/backend/distributed/operations/repair_shards.c

```diff
@@ -1498,15 +1501,10 @@ UpdateColocatedShardPlacementMetadataOnWorkers(int64 shardId,
 		ShardInterval *colocatedShard = (ShardInterval *) lfirst(colocatedShardCell);
 		StringInfo updateCommand = makeStringInfo();
 
-		appendStringInfo(updateCommand, "UPDATE pg_dist_shard_placement "
-										"SET nodename=%s, nodeport=%d WHERE "
-										"shardid=%lu AND nodename=%s AND nodeport=%d",
-						 quote_literal_cstr(targetNodeName),
-						 targetNodePort,
+		appendStringInfo(updateCommand,
```

Comment:
Updating a view `pg_dist_shard_placement` sounds confusing and could even be wrong after `citus_update_node`. That's why I switched using updating the underlying `pg_dist_placement`